### PR TITLE
Update dockerfile alert extraction

### PR DESCRIPTION
**Pull request by jzelenjak**
Updates Dockerfile to copy extracted attack stages, alert signatures and mappings from the main branch.

NB! This PR closes issue https://github.com/tudelft-cda-lab/SAGE/issues/17 and has to be merged after PR https://github.com/tudelft-cda-lab/SAGE/pull/19.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,8 +12,9 @@ COPY requirements.txt /root
...
 && mkdir input/
 
-#COPY alerts/*.json /root/input/ 
-#RUN chmod -R a+rw /root/input/
+# Make sure to put alert files into the directory
+COPY alerts/*.json /root/input/
+RUN chmod -R a+rw /root/input/
 
 
 ##########################################
```
Comment (jzelenjak) [inline Dockerfile:20]: These two commented commands below (lines 23, 25)... I assume they were supposed to update and upgrade the system. Should they remain commented for now?

Docker build logs mention at some point that a newer pip version is available.

EDIT: when running *with* those commands, the image seems to build fine.

Comment (azqanadeem) [reply Dockerfile:20]: Let's uncomment it. Maybe takes longer to build but at least it will be updated

**Review by azqanadeem — CHANGES_REQUESTED** [review]
Uncomment the image updates lines (see my reply on your question)

File: Dockerfile (hunk 1 above)
Comment (jzelenjak) [reply Dockerfile:20]: Done

**Comment by jzelenjak** [comment]
I have uncommented those lines. The image builds fine with the current version on the main branch, the container also runs without problems

**Review by azqanadeem — APPROVED** [review]

**Pull request merged**